<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="cadastroPage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 10px;
color: white;
background-color: #8FA2DA;
}
QPushButton:pressed {
background-color: lightblue;
}
</string>
     </property>
     <property name="text">
      <string>Cadastro</string>
     </property>
    </widget>
    <widget class="QPushButton" name="excluirPage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 2px;
background-color: #4FCFB4;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>Excluir</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarPage">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 2px;
background-color: #4FCFB4;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Qtd:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="salvar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #3CD873;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QPushButton" name="desc">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #8FA2DA;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Desc</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="asc">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #8FA2DA;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Asc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="organizar">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #8FA2DA;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Organizar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomeIn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 10px;
border-style: none;
background-color: #B2CDE2;
color: #523E3E;
}</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="qtdIn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
border-radius: 10px;
border-style: none;
background-color: #B2CDE2;
color: #523E3E;
}</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idIn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 10px;
background-color: #B2CDE2;
color: #523E3E;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excluir">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #F26A1D;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="excluirTudo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #F26A1D;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Excluir tudo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomeBuscarIn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
border-radius: 10px;
background-color: #B2CDE2;
color: #523E3E;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
border-radius: 10px;
background-color: #8FA2DA;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="doado">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font: 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>Doado:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1161</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(45, 177, 118);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Arial&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Dados</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gerarExcel">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 2px;
background-color: #4FCFB4;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>Gerar Excel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gerarPdf">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 2px;
background-color: #4FCFB4;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>Gerar PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gerarGrafico">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 2px;
background-color: #4FCFB4;
color: white;
}
QPushButton:pressed {
background-color: lightblue;
}</string>
     </property>
     <property name="text">
      <string>Gerar Gráfico</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>61</width>
      <height>211</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>1151</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4FCFB4;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="por_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 23pt &quot;Segoe UI&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="total">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>230</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 23pt &quot;Segoe UI&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLabel" name="por_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>170</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 23pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Média:</string>
     </property>
    </widget>
    <widget class="QLabel" name="media">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 23pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Media</string>
     </property>
    </widget>
    <widget class="QLabel" name="por_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 23pt &quot;Segoe UI&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Arrecadado:</string>
     </property>
    </widget>
    <widget class="QLabel" name="arrecadado">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 23pt &quot;Segoe UI&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Arrecadado</string>
     </property>
    </widget>
    <widget class="QLabel" name="por">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 23pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Por</string>
     </property>
    </widget>
    <widget class="QLabel" name="por_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font: 23pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Meta:            R$2000</string>
     </property>
    </widget>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>Pages</zorder>
   <zorder>line</zorder>
   <zorder>table</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
